vizro-core/hatch.toml

```
@@ -111,6 +111,8 @@ build = "mkdocs build --strict"
 # Disable warnings on the linkcheck so that HTTP redirects are accepted. We could ignore only that warning and specify
 # more advanced settings using a linkcheckerrc config file.
 link-check = "linkchecker site --check-extern --no-warnings --ignore=404.html --ignore-url=127.0.0.1 --ignore-url=https://vizro.readthedocs.io/ --ignore-url=https://platform.ploomber.io/register --ignore-url=https://huggingface.co/join"
+# Check if link to QB logo is not deleted from the docs.
+logo-check = "grep -q 'logo_watermark_extended.svg' docs/pages/user-guides/install.md || echo exit 1"
```

```suggestion
logo-check = "grep -q 'logo_watermark_extended.svg' docs/pages/user-guides/install.md"
```

To test it to be sure (remove the link and push and see what the docs build does). But `grep` will return exit code 1 if no match is found, which should then fail the build.

If that doesn't work then I think you mean `exit 1` without the `echo` anyway.

---

Done